### DISCUSSION TITLE
[3.50] Debug Graphics for Isometric Tilemaps renderDebug()

### DISCUSSION THREAD

**Discussion by ZaDarkSide** [photonstorm/phaser#5915] [Feature Requests]
## Version

* Phaser Version: Phaser v3.50.0-beta.11

* Operating system: Windows 10

* Browser: Chrome 87.0.4280.88

## Description

When using isometric tilemaps the debug for them renders square tiles instead of isometric squares.

## Example Test Code

```js
let debug = this.add.graphics().setAlpha(0.07).setDepth(20);
let map = this.add.tilemap('map');
let tileset = map.addTilesetImage('isometric_grass_and_water', 'tiles');
let layer1 = map.createLayer('Tile Layer 1', [tileset]);
layer1.setCollisionByProperty({ collision: true });

map.renderDebug(debug, {
	tileColor: new Phaser.Display.Color(40, 255, 48, 255),
	collidingTileColor: new Phaser.Display.Color(243, 134, 48, 255),
	faceColor: new Phaser.Display.Color(40, 39, 37, 255)
})
```

## Additional Information

![image](https://user-images.githubusercontent.com/291278/101989007-1802b380-3ca6-11eb-8f7b-3b4b814bf71d.png)

**Comment by photonstorm** [comment]
I've changed the code so it won't try to debug a map unless it's orthogonal. I'll leave this issue open as a future feature request, however.
